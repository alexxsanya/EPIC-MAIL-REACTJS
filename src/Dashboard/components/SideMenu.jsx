import React from 'react';
import {
  Nav,
} from 'react-bootstrap';

const SideMenu = () => (
  <Nav defaultActiveKey="/home" className="flex-column">
    <Nav.Link href="/dashboard/inbox">Inbox Messages</Nav.Link>
    <Nav.Link href="/dashboard/sent">Sent Messages</Nav.Link>
  </Nav>
);
export default SideMenu;
