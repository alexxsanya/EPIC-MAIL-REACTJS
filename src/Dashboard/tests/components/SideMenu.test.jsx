/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SideMenu from '../../components/SideMenu';

describe('SideMenu Component', () => {
  const component = shallow(<SideMenu />);

  it('should match the snapshot', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
