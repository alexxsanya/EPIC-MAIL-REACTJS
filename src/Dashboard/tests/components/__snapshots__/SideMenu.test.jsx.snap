// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu Component should match the snapshot 1`] = `
<Nav
  className="flex-column"
  defaultActiveKey="/home"
  fill={false}
  justify={false}
>
  <NavLink
    as={[Function]}
    disabled={false}
    href="/dashboard/inbox"
  >
    Inbox Messages
  </NavLink>
  <NavLink
    as={[Function]}
    disabled={false}
    href="/dashboard/sent"
  >
    Sent Messages
  </NavLink>
</Nav>
`;
